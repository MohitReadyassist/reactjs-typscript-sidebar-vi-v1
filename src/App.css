* {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}

html,
body {
    color: #fff;
}

.overview {
    display: flex;
    justify-content: center;
    align-items: center;
    font-size: 3rem;
    color: #000000;
    height: 90vh;
}
.Heading{
    margin-left: 650px;
}
.Header{
    margin-left: 400px;
}