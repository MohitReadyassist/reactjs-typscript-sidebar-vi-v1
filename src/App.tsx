import React, { FC } from 'react';
import './App.css';
import Sidebar from './components/Sidebar';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import { CertifiedUserList, Homes, CourseList, TutorList } from './pages/Overview';




const App: FC = () => {
    return (
        <Router>
            <Sidebar />
            <Switch>

                <Route path="/Homes" component={Homes} exact>{Homes}</Route>
                <Route path="/overview/TutorList" component={TutorList} exact></Route>
                <Route path="/overview/CourseList" component={CourseList} exact></Route>
                <Route path="/overview/CertifiedUserList" component={CertifiedUserList} exact></Route>
            </Switch>
        </Router>

    );
};

export default App;
