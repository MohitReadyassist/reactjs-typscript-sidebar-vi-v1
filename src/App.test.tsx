import React from 'react';
import { render } from '@testing-library/react';
import App from './App';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer'

test("First snapshot test", () => {
  const component = renderer.create(
    <App />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})



it('renders without crashing', () => {
  const setUp = (initialState = {}) => {
    const wrapper = shallow(<App />);
    return wrapper;
  }
});

test.skip('renders learn react link', () => {
  const { getByText } = render(<App />);


});
