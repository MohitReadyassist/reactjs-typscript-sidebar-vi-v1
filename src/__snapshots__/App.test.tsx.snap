// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First snapshot test 1`] = `
<div>
  <div
    className="sc-hKgILt fudpPp"
  >
    <a
      className="sc-jSgupP kClYHy"
      href="/"
      onClick={[Function]}
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": "#fff",
          }
        }
        viewBox="0 0 1024 1024"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"
        />
      </svg>
    </a>
    <div
      className="Heading"
    >
      <p>
        Learning and Certification
      </p>
    </div>
    <div
      className="Header"
    >
      <p>
        Hii, hr@utthunga.com
      </p>
    </div>
  </div>
  <div
    className="sc-eCssSg cbntSp"
  >
    <span>
      <h3>
        Admin Home
      </h3>
    </span>
    <div
      className="sc-gKsewC"
    >
      <a
        className="sc-jSgupP kClYHy"
        href="/"
        onClick={[Function]}
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": "#fff",
            }
          }
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M563.8 512l262.5-312.9c4.4-5.2.7-13.1-6.1-13.1h-79.8c-4.7 0-9.2 2.1-12.3 5.7L511.6 449.8 295.1 191.7c-3-3.6-7.5-5.7-12.3-5.7H203c-6.8 0-10.5 7.9-6.1 13.1L459.4 512 196.9 824.9A7.95 7.95 0 0 0 203 838h79.8c4.7 0 9.2-2.1 12.3-5.7l216.5-258.1 216.5 258.1c3 3.6 7.5 5.7 12.3 5.7h79.8c6.8 0 10.5-7.9 6.1-13.1L563.8 512z"
          />
        </svg>
      </a>
      <a
        className="sc-bdfBwQ eYwdxL"
        href="/Homes"
        onClick={[Function]}
      >
        <div>
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": "#fff",
              }
            }
            viewBox="0 0 1024 1024"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M946.5 505L560.1 118.8l-25.9-25.9a31.5 31.5 0 0 0-44.4 0L77.5 505a63.9 63.9 0 0 0-18.8 46c.4 35.2 29.7 63.3 64.9 63.3h42.5V940h691.8V614.3h43.4c17.1 0 33.2-6.7 45.3-18.8a63.6 63.6 0 0 0 18.7-45.3c0-17-6.7-33.1-18.8-45.2zM568 868H456V664h112v204zm217.9-325.7V868H632V640c0-22.1-17.9-40-40-40H432c-22.1 0-40 17.9-40 40v228H238.1V542.3h-96l370-369.7 23.1 23.1L882 542.3h-96.1z"
            />
          </svg>
          <span
            className="sc-gsTCUz sRnIy"
          >
            Homes
          </span>
        </div>
        <div>
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": "#fff",
              }
            }
            viewBox="0 0 1024 1024"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
            />
          </svg>
        </div>
      </a>
      <a
        className="sc-bdfBwQ eYwdxL"
        href="/overview/CertifiedusersList"
        onClick={[Function]}
      >
        <div>
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": "#fff",
              }
            }
            viewBox="0 0 640 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M423.3 440.7c0 25.3-20.3 45.6-45.6 45.6s-45.8-20.3-45.8-45.6 20.6-45.8 45.8-45.8c25.4 0 45.6 20.5 45.6 45.8zm-253.9-45.8c-25.3 0-45.6 20.6-45.6 45.8s20.3 45.6 45.6 45.6 45.8-20.3 45.8-45.6-20.5-45.8-45.8-45.8zm291.7-270C158.9 124.9 81.9 112.1 0 25.7c34.4 51.7 53.3 148.9 373.1 144.2 333.3-5 130 86.1 70.8 188.9 186.7-166.7 319.4-233.9 17.2-233.9z"
            />
          </svg>
          <span
            className="sc-gsTCUz sRnIy"
          >
            CertifiedusersList
          </span>
        </div>
        <div />
      </a>
    </div>
  </div>
</div>
`;
